Rakefile

```diff
@@ -196,22 +197,22 @@ namespace :toolchain do
   desc 'Sync Rust toolchain to all sources'
   task sync: %i[sync:manifests sync:ci]
 
-  rust_toolchain = File.read('rust-toolchain').chomp
+  rust_toolchain = TOML.load_file('rust-toolchain.toml')
+  toolchain_version = rust_toolchain['toolchain']['channel']
```

Comment:
this is great!

Reply:
I did some ol `symbolize_keys` in tomlrb. Thoughts?

Reply:
sweet, I like it.